### PR TITLE
VBLOCKS-1067: Added  sdk version override

### PR DESCRIPTION
**Pull request by afalls-twilio**
## Submission Checklist

 - [ ] Updated the `CHANGELOG.md` to reflect any **feature**, **bug fixes**, or **known issues** made in the source code
 - [ ] Tested code changes and observed expected behavior in the example app
 - [ ] Performed a visual inspection of the `Files changed` tab prior to submitting the pull request for review to ensure proper usage of the style guide

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

## Description

Added the ability to override the SDK version for insights publishing

## Breakdown

- Modified such that version is overridden by package.json
- Made it use Constants.src so that its a platform neutral way.

## Validation

- None

### CODE REVIEW

**Review by bobiechen-twilio — APPROVED** [review]
Nice!

**Review by charliesantos — APPROVED** [review]
This is great!

File: scripts/constants.js [hunk 1]
```diff
@@ -167,9 +167,28 @@ async function transform(constantsPath, templatePath, outputPath) {
...
+async function substituteVersion(constantsPath) {
+    const packageJsonPath = './package.json';
+    const packageJsonSource = (await readFile(packageJsonPath)).toString('utf-8');
+
+    const json = JSON.parse(packageJsonSource);
+    const version = json.version;
+
+    const constantsInput = (await readFile(constantsPath)).toString('utf-8');
+    const constantsOutput = constantsInput.replace(
+        /(ReactNativeVoiceSDKVer\s*=\s*)(.*)/g,
```
Comment (mhuynh5757) [inline scripts/constants.js:179]:
```suggestion
        /^(ReactNativeVoiceSDKVer=)(.*)$/g,
```

Comment (afalls-twilio) [reply scripts/constants.js]: why?

Comment (mhuynh5757) [reply scripts/constants.js]: After discussing in Slack, we've decided that we will leave the regex flexible in this PR, and then making validators down the line and make this regex more strict.

**Review by mhuynh5757 — APPROVED** [review]

**Pull request merged**